src/parser/jobs/pld/modules/Tincture.tsx

```diff
@@ -33,18 +33,21 @@ export class Tincture extends CoreTincture {
 
 		this.addEvaluator(new ExpectedActionsEvaluator({
 			expectedActions: [
-				{
-					action: this.data.actions.EXPIACION,
-					expectedPerWindow: 1,
-				},
-				{
-					action: this.data.actions.CIRCLE_OF_SCORN,
-					expectedPerWindow: 1,
-				},
+				{action: this.data.actions.EXPIACION, expectedPerWindow: 1},
+				{action: this.data.actions.CIRCLE_OF_SCORN, expectedPerWindow: 1},
+				{action: this.data.actions.GORING_BLADE, expectedPerWindow: 1},
+				{action: this.data.actions.CONFITEOR, expectedPerWindow: 1},
+				{action: this.data.actions.BLADE_OF_FAITH, expectedPerWindow: 1},
+				{action: this.data.actions.BLADE_OF_TRUTH, expectedPerWindow: 1},
+				{action: this.data.actions.BLADE_OF_VALOR, expectedPerWindow: 1},
+				{action: this.data.actions.HOLY_SPIRIT, expectedPerWindow: 1},
 			],
 			suggestionIcon,
 			suggestionContent: <Trans id="pld.tincture.suggestions.trackedactions.content">
-				One use of <ActionLink {...this.data.actions.EXPIACION}/> and at least one use of <ActionLink {...this.data.actions.CIRCLE_OF_SCORN}/> should occur during every <ActionLink {...this.data.actions.INFUSION_STR}/> window.
+				Try to land at least one cast of <DataLink action="EXPIACION" />, <DataLink action="CIRCLE_OF_SCORN" />
```

Comment:
this really doesn't read well i don't think - would be good to cut out the bulk of the big-list-of-actions, as it just becomes a wall of text and people glaze over.

this is just crap i'm thinking up off the top of my head, but something to the tune of

"try to land the [confetior] combo and other key damage abilities within [tincture] windows. check the [link to tinctures module] for details"

would help - i think the table in the module is much more clear around expectations than written text.

Reply:
Do you have any "link to modules" examples? I would love to use that more! That being said, I do think this is subjective and it's hard for me to justify not being clear in the notes before graphing it out in the module. (I think linking to the module is a fantastic suggestion though)

Reply:
`this.parser.scrollTo(handle)`

there's an example in `core/modules/Weaving` (for much the same purpose, linking to the module for more detail).